server/player/player.go

```diff
@@ -964,6 +989,26 @@ func (p *Player) Respawn() {
 	p.SetVisible()
 }
 
+// SpawnPos returns spawn position in the current world.
+func (p *Player) SpawnPos() cube.Pos {
+	return p.World().PlayerSpawn(p.uuid)
+}
+
+// SetSpawnPos sets spawn position for the player in the provided world.
+func (p *Player) SetSpawnPos(pos cube.Pos, w *world.World) {
+	w.SetPlayerSpawn(p.UUID(), pos)
+}
```

Comment:
I also don't see a reason why these methods need to exist if the caller already has access to the player and world? They can just call PlayerSpawn/SetPlayerSpawn directly on the world

Reply:
SetSpawnPos used here: https://github.com/df-mc/dragonfly/pull/938/files#diff-b1e7a7f21487969b0d5980153a20003b2e51eecd41a8aab7d1215f9432cbad40R118
and here: https://github.com/df-mc/dragonfly/pull/938/files#diff-0c77a3c2d3771dbd25a7b738b13d7f35d3871ad3a5f241c192c5fc242eea02e5R58

Reply:
Yes you can just do w.SetPlayerSpawn(u.UUID(), pos) ?